### ISSUE TITLE
Change list to table for easier scanning

### ISSUE THREAD

**Issue by bradleycamacho** [newrelic/docs-website#5981]
This ticket is part of the XD bug bash, Athena initiative, and FutureStack quality work.

# Description
This doc has an unordered-list that is a bit hard to parse and scan. We should transform it into a table for easier scanning.

## Doc
https://docs.newrelic.com/docs/apm/apm-ui-pages/monitoring/agent-specific-ui-pages

## Action items
- [ ] Remove the list and transform it into a table

**Comment by pizzaz93** [comment]
I can't seem to find the source for this page, is it a deleted cached page?
